src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -142,8 +143,9 @@ private async void NextButton_Click(object sender, RoutedEventArgs e)
                     var serverUri = ToUri(ServerComboBox.Text);
 
                     // block clicking "next" until login request is done
-                    ToggleLoading(true);
-                    await AzureDevOps.HandleLoginAsync(CredentialPromptType.PromptIfNeeded, serverUri).ConfigureAwait(false);
+                    ToggleLoading(true, () => Task.Run(async () => {
```

Comment:
idea is to wrap minimal parts of the code in a blocking dialog without touching the underlying workflow